tests/internal/py/fdk.py

```
@@ -0,0 +1,1114 @@
+                    f"unreachable: no graphql type found for variant {variant_ty}"
+                )
+            gql_ty = gql_ty.strip("!")
+
+            sub_node_list += f"... on {gql_ty} {{ "
+            for node in sub_nodes:
+                sub_node_list += f"{convert_query_node_gql(ty_to_gql_ty_map, node, variables, files)} "
+            sub_node_list += "}"
+        out += f" {{ {sub_node_list}}}"
+    elif isinstance(node.sub_nodes, list):
+        sub_node_list = ""
+        for node in node.sub_nodes:
```

Ruff (0.8.2)

Loop control variable `node` overrides iterable it iterates

(B020)